src/nyx/features/glcm.cpp

```diff
@@ -232,7 +255,8 @@ void GLCMFeature::calculateCoocMatAtAngle(
 	PixIntens max_val, 
 	bool normalize)
 {
-	matrix.allocate(n_levels, n_levels, 0.0); 
+	matrix.allocate(n_levels, n_levels); 
+	std::fill(matrix.begin(), matrix.end(), 0.);

```

Comment:
std::fill is called twice (line 259 and 261)

Reply:
Good eye Jesse